apps/services/utils.py

```diff
@@ -0,0 +1,84 @@
+from apps.common.exceptions import BackendException
+from apps.users.services import student_list_service
+
+from collections import defaultdict
+from fastapi import UploadFile, status as http_status
+from sqlalchemy.ext.asyncio import AsyncSession
+from typing import Callable, DefaultDict, Dict, List, Set, Tuple
+import xlrd
+
+
+def check_file_content_type(file: UploadFile) -> None:
+    if file.content_type != "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet":
+        raise BackendException(
+            message="Uploaded file have invalid type.",
```

Comment:
I wanted to try this checking, but i can't understand what type should I use
(I tried to add  **_hostel_booking_template.docx_** from our project) 
or maybe you can add some file to our group to check how it works)

Reply:
.xlsx

Reply:
resolved